src/main/java/com/amplitude/Event.java

```diff
@@ -9,56 +9,162 @@
 
 public class Event {
 
+  /** A unique identifier for your event. */
   public String eventType;
+  /**
+   * A readable ID specified by you. Must have a minimum length of 5 characters. Required unless
+   * device_id is present.
+   */
   public String userId;
+  /**
+   * A device-specific identifier, such as the Identifier for Vendor on iOS. Required unless user_id
+   * is present. If a device_id is not sent with the event, it will be set to a hashed version of
+   * the user_id.
+   */
   public String deviceId;
 
+  /**
+   * The timestamp of the event in milliseconds since epoch. If time is not sent with the event, it
+   * will be set to the request upload time.
+   */
   public long timestamp = System.currentTimeMillis();
+  /** The current Latitude of the user. */
   public double locationLat;
+  /** The current Longitude of the user. */
   public double locationLng;
 
+  /** The current version of your application. */
   public String appVersion;
+
```

Comment:
missing comments for `versionName`?

Reply:
I couldn't actually find documentation for this variable? @yuhao900914 ?

Reply:
I did add the version_name in the nodejs even type. But I cannot find where it comes from now. Will check.

Reply:
Just discuss this with @kelvin-lu, since this also exists in node js sdk. The version name does not exist in current http v2 documents, so that's might be the legacy code.